### PR TITLE
APB rename

### PR DESCRIPTION
**Pull request by johnkim76**
change 'ansibleapps' to apb

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -42,25 +42,25 @@ parameters:
...
 description | string | yes | | A human readable description of the parameter.
 type | string | yes | | Type of parameter. `bool`, `int`, `float`, `string`, are valid
 default | bool,int,float,string|  no | | An optional default value for the parameter.
 
 
-## Running AnsibleApps
+## Running APB
 
-AnsibleApps are containers that are run with the following command:
+APB are containers that are run with the following command:
```
Comment (jwmatthews) [inline docs/design.md]: APBs

File: docs/design.md [hunk 2]
```diff
@@ -42,25 +42,25 @@ parameters:
...
 
 * `container_name`
 
-The name of the ansibleapp container, i.e. `ansibleapp/etherpad`
+The name of the APB container, i.e. `apb/etherpad`
 
 * `action`
 
-One of the 4 possible actions an ansible app can take. At a minimum, an ansibleapp
+One of the 4 possible actions an ansible app can take. At a minimum, an APB
```
Comment (jwmatthews) [inline docs/design.md]: remove "ansible app"

**Comment by jwmatthews** [comment]
This PR is _just_ addressing the name change.
We'll need to revisit the design docs and update later.

ACK

**Comment by jmrodri** [comment]
You guys should've looked at the apb branch first. I did a lot of this already.

https://github.com/fusor/ansible-playbook-bundle/tree/apb

**Pull request merged**